### PR TITLE
Add progress bar tests

### PR DESCRIPTION
**Pull request by anikielx**
Signed-off-by: Adriana Nikelska <adrianax.nikielska@intel.com>

### CODE REVIEW

File: test/functional/tests/progress_bar/test_progress_bar_during_io.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+               .read_write(ReadWrite.write)
+               .block_size(Size(1, Unit.Blocks4096))
+               .direct()
+               .io_engine(IoEngine.libaio))
+        for i, core in enumerate(cores):
+            fio.add_job(f"core{i}").target(core.path)
+        fio_pid = fio.run_in_background()
+
+        TestRun.LOGGER.info("Wait 8 minutes.")
+        sleep(480)
```
[the remark this answers is not in the document]
Comment (anikielx) [reply test/functional/tests/progress_bar/test_progress_bar_during_io.py:58]: Fio is running in background so it is necessary to wait some time for data. It doesn't work with sync() only.

**Review by mmichal10 — APPROVED** [review]

**Comment by mmichal10** [comment]
Retest this please

**Comment by mmichal10** [comment]
ok to test

**Comment by mmichal10** [comment]
Retest this please

**Comment by katlapinka** [comment]
Please update headers in each file with correct date

**Comment by anikielx** [comment]
> Please update headers in each file with correct date

I checked it and all headers are correct

**Review by katlapinka — APPROVED** [review]

**Pull request merged**